.eslintignore

```diff
@@ -1,11 +1,9 @@
-bower_components/*
 experiments/*
 node_modules/*
 bin/*
 spec/fixtures/*
 spec/support/*
 LICENSE
-bower.json
 package.json
 README.md
 npm-debug.log
```

Comment:
Not relevant to this PR, but you only need to add js files (and jsx, if you specify --ext js,jsx) - ESLint ignores everything else by default